apps/system/utils/kdbg_stackmonitor.c

```diff
@@ -154,11 +154,12 @@ static void stkmon_print_active_values(char *buf)
 	int i;
 	stat_data stat_info[PROC_STAT_MAX];
 
-	stat_info[0] = strtok(buf, " ");
+	stat_info[0] = buf;
 
-	for (i = 1; i < PROC_STAT_MAX; i++) {
-		stat_info[i] = strtok(NULL, " ");
+	for (i = 0; i < PROC_STAT_MAX - 1; i++) {
```

Comment:
This change is mismatched with your description
```
1. fix bug about loop iteration
The number of stat info is PROC_STAT_MAX.
A loop for tokenizing should iterate (PROC_STAT_MAX-1) times.
```
There is no change on number of iterate.

Reply:
Yes sunghan is right , it is still PROC_STAT_MAX,
Earlier it was missing the first (0) index.

Reply:
I modified commit message